addons/website_forum/controllers/website_forum.py

```diff
@@ -366,12 +366,13 @@ def question_reopen(self, forum, question, **kwarg):
 
     @http.route('/forum/<model("forum.forum"):forum>/question/<model("forum.post"):question>/delete', type='http', auth="user", methods=['POST'], website=True)
     def question_delete(self, forum, question, **kwarg):
-        question.active = False
-        return request.redirect("/forum/%s" % slug(forum))
+        question.state = 'deleted'
+        return request.redirect("/forum/%s" % (slug(forum)))
```

Comment:
why this change ?

Reply:
because we don't want the moderator to archive forums, instead put them in "deleted" state. we decided that archiving the forum would only be happening from back-end.

Reply:
I was asking for the `()` around slug :)